### DISCUSSION TITLE
Issue whilst publishing Power App custom component

### DISCUSSION THREAD

**Discussion by jp632** [microsoft/powerplatform-build-tools#1009] [General]
Import Solution Failed: CustomControl with name SampleNameSpace.EditablePdf failed to import with error: Principal user (Id=********-****-****-****-************, type=8, roleCount=5, privilegeCount=1478, accessMode='0 Read-Write', AADObjectId='********-****-****-****-************', MetadataCachePrivilegesCount=5848, businessUnitId=********-****-****-****-************), is missing prvCreateCustomControl privilege (Id=********-****-****-****-************) on OTC=9753 for entity 'customcontrol' (LocalizedName='Custom Control'). context.Caller=********-****-****-****-************. Consider adding missed privilege to one of the principal (user/team) roles.

**Comment by mannybellas** [comment]
Doesn't appear to be a defect with `pac`. The error message tells you that your current user doesn't not have the prvCreateCustomControl privilege
